heart/heartbeat.py

```diff
@@ -116,6 +120,15 @@ def __init__(self, ct, reactor, server, sockfd):
 		self.sockfd.setblocking(False)
 		self.reactor.push(ct, self.update)
 
+	def sanetize_str(self, string):
```

Comment:
You spelt "sanitise"/"sanitize" wrong. I'd go with "sanitise" since that's used in the main project.